tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -10,7 +10,7 @@
       "zone": "{{ .zone }}",
       {{- end }}
       {{- if .sub_zone }}
-      "sub_zone": "{{ .sub_zone }}",
```

Comment:
if you don't have a subzone you will still get a trailing `,`

Reply:
thx,fixed it.